### ISSUE TITLE
Is it possible to get the jar for this?

### ISSUE THREAD

**Issue by viocu** [BigScary/GriefPreventionFlags#1]
I can't seem to compile it (i'm a newb), so is it possible to just get the jar? Thanks!

**Comment by MCTyler** [comment]
Some guy re-coded it and put it on Spigot.

**Comment by ShaneBeee** [comment]
That would be me, I popped it up on spigot for anyone who is wanting it!
https://www.spigotmc.org/resources/griefprevention-flags.55773/

I personally do not know anything about java .... but im going to start searching for someone to help me update this plugin, apparently theres a PVP issue

**Comment by MCTyler** [comment]
There is also this....https://www.spigotmc.org/resources/worldguard-extra-flags.4823/

**Comment by ShaneBeee** [comment]
Yes, but that is for world guard regions, and cant be used on GP regions
